### DISCUSSION TITLE
How can I avoid cookies persisting on a single resty Client?

### DISCUSSION THREAD

**Discussion by joe-cranney** [go-resty/resty#456] [Q&A] [answered]
I'm working on a specific application where I would like to just have a single, global, request client. The client will make several different requests. However, I noticed that cookies returned from one request on the client will persist on a subsequent request. When I try to list the clients cookies, it appears to be empty, yet the cookies are still submitted. How can I avoid this without creating an entirely new client for each request?

EDIT: I'm not using a cookie jar, yet they still persist. Seems like a weird bug.

**Comment by jeevatkm** [comment, marked as answer]
@jj0e Typically, when you use `resty.New()` to create a resty client, it initializes with cookie jar.  I think for your use case, create your HTTP client instance and then use `resty.NewWithClient(...)`
